pulp_rpm/app/models.py

```diff
@@ -1007,6 +1094,26 @@ def grplist_to_lst(cls, value):
                              'default': i.default})
         return grp_list
 
+    @classmethod
+    def list_to_idlist(cls, list):
```

Comment:
Up to you. I just noticed that this is defined on the other class as well.
In theory, it can go to comps.py as helper function, just not to duplicate code twice.
It's just few lines and quite straightforward so not critical and up to you.

Reply:
moved it to comps.py as helper function